examples/shardingsphere-jdbc-example/single-feature-example/sharding-example/sharding-raw-jdbc-example/src/main/resources/META-INF/sharding-databases-range.yaml

```diff
@@ -58,6 +58,7 @@ rules:
     - t_address
   defaultDatabaseStrategy:
     standard:
+      shardingColumn: user_id
```

Comment:
Hi, I think it is better to use `defaultShardingColumn` here, need to modify the code in `ShardingRule`.

Reply:
Ok, i resubmitted a pr.